README.md

```diff
@@ -114,6 +114,16 @@ Here is the example data you could pass into the tree props.
 	}
 ];
 ```
+#### metadata
```

Comment:
i agree with @conorhastings. i would like to see documentation added for this new prop